### PR TITLE
add hw_1

### PR DESCRIPTION
**Pull request by nico-nico13**

### CODE REVIEW

File: src/test/java/com/epam/tc/hw1/DivideTest.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import org.testng.annotations.Test;
+
+
+public class DivideTest {
+
+    @Test(
+            dataProviderClass = DataProviderDivide.class,
+            dataProvider = "testing divide numbers"
+    )
+    public void testDivide(int a, int b, int expected) {
```
Comment (akulebyakin) [inline src/test/java/com/epam/tc/hw1/DivideTest.java:14]: Let's add some test that will check division by 0

File: src/test/resources/AddSubtract.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
+<suite name="AddSubtract">
+    <test verbose="2" preserve-order="true" name="C:\Users\EVA Unit-11\Desktop\EPAM\selenium\HW1. Unit Frameworks\HW1">
```
Comment (akulebyakin) [inline src/test/resources/AddSubtract.xml:4]: Why such a strange name for a test?
`name="C:\Users\EVA Unit-11\Desktop\EPAM\selenium\HW1. Unit Frameworks\HW1"`

**Review by akulebyakin — CHANGES_REQUESTED** [review]
Please review comments

**Comment by nico-nico13** [comment]
@akulebyakin Добавил отдельный тест на деление на ноль

**Review by akulebyakin — APPROVED** [review]
approved

**Pull request merged**